package.json

```diff
@@ -138,7 +138,7 @@
 		"@apollo/react-testing": "^3.1.3",
 		"@babel/plugin-proposal-class-properties": "^7.8.3",
 		"@babel/plugin-transform-runtime": "^7.8.3",
-		"@babel/preset-env": "^7.8.6",
+		"@babel/preset-env": "^7.9.0",
```

Comment:
The error was saying:
```
 Could not find plugin "proposal-numeric-separator". Ensure there is an entry in ./available-plugins.js for it. (While processing: "/home/travis/build/eventespresso/event-espresso-core/node_modules/babel-preset-react-app/index.js$0")
```
and I've updated this package to latest master where that plugin exists: 
https://github.com/babel/babel/blob/master/packages/babel-preset-env/src/available-plugins.js#L17